tests/test_amclient.py

```diff
@@ -17,6 +17,7 @@
 SS_URL = 'http://192.168.168.192:8000'
 AM_USER_NAME = 'test'
 AM_API_KEY = '3c23b0361887ace72b9d42963d9acbdf06644673'
+AM_API_KEY_2 = 'ff8fa7f316168508bf9bc743ac589ec55bfd611a'
```

Comment:
Instead of adding a second testing key, you can modify the recorded cassette to use the existing one.  

Alternately, we could configure vcr.py not to record the api key, to avoid problems like this, and update the cassettes to remove the api_key param.  eg https://vcrpy.readthedocs.io/en/latest/advanced.html#filter-information-from-http-querystring